src/ui/dom/dom.js

```diff
@@ -144,6 +144,19 @@ export default class DOM {
     }
   }
 
+  static removeClass (node, className) {
```

Comment:
i know this is a copy of `addClass` but can we add jsdocs and turn our `let`s into `const`s when we can?

Reply:
sure!